pyanalyze/stacked_scopes.py

```diff
@@ -1144,7 +1144,7 @@ def loop_scope(self) -> Iterator[SubScope]:
         with self.subscope() as main_scope:
             loop_scopes.append(main_scope)
             with qcore.override(self, "current_loop_scopes", loop_scopes):
-                yield main_scope
+                yield loop_scopes
```

Comment:
The annotation on line 1142 is wrong now, it should be `Iterable[Sequence[Subscope]]` I think.

For some reason this failed the self check only on 3.6.